cloudhub-1/modules/ROOT/Edits/vpn-maintenance.adoc

```diff
@@ -0,0 +1,71 @@
+* An Anypoint VPN tunnel pair configured with Static routing incurs some downtime before reconnection. The reconnection takes longer if you do not have an automated failover mechanism to detect failed tunnels.
+* For an Anypoint VPN configured without BGP routing or a failover mechanism or with only one tunnel configured, the outage is up to 60 minutes long while the update occurs and the original tunnel comes back up.
+
+== Troubleshoot Issues During an Anypoint VPN On-demand Maintenance
+
+These are some of the possible issues you may encounter during the On-demand Maintenance:
+
+=== Tunnels not Coming Up
+
+If the tunnels are not back up within 10-15 minutes at any point in time, try restarting your end of the tunnels to bring them back up.
+
+If your VPN is configured with static routing, try generating 'interesting' traffic towards the Anypoint VPN to bring up the VPN. See https://help.mulesoft.com/s/article/How-to-Generate-Interesting-Traffic-for-Anypoint-VPN[How to Generate Interesting Traffic for Anypoint VPN^] for more information.
+
+=== Tunnel is Going Down Due to Inactivity
+
+If there is no traffic over the VPN tunnel, it goes down due to inactivity. In this case, generate interesting traffic. We recommend you move towards BGP.
+
+=== IKE Phase Lifetimes Mismatch
+
+==== If You Are Using IKEv2:
+
+To maintain a reliable tunnel operation after the on-demand maintenance, ensure that IKEv2 Phase 1 and Phase 2 lifetimes are set as described in xref:vpn-about.adoc#ipsec-settings[the documentation]:
+
+* IKE Phase 1 lifetime: 28000 seconds
+* IKE Phase 2 lifetime: 3000 second
+
+[NOTE]
+AWS documentation states 28800 seconds for IKE Phase 1 lifetime, and 3600 seconds for IKE Phase 2 lifetime. MuleSoft recommends the 28000/3000 combination for your firewall device to timeout first and control the rekey process.
+
+==== If You Are Using IKEv1:
+
+If you are using IKEv1 please use the default configuration: 28800 seconds for Phase 1 lifetime, and 3600 seconds for Phase 2 lifetime.
+
+=== Anypoint VPN Goes Down Periodically Due to PFS not Enabled for IPsec Phase 2
+
+If you are using Azure VPN, ensure that PFS is enabled and adjust lifetimes for Phase 2. For more information, see:
+
+* https://help.mulesoft.com/s/article/Anypoint-VPN-goes-Down-Periodically-Due-to-PFS-is-not-Enabled-for-IPsec-Phase-2[Anypoint VPN Goes Down Periodically Due to PFS not Enabled for IPsec Phase 2^]
+* https://help.mulesoft.com/s/article/Anypoint-VPN-to-Azure-keeps-going-down-every-hour[Anypoint VPN to Azure Keeps Going Down Every Hour^]
+
+=== Anypoint VPN with BGP Routing Enters Asymmetric Mode after Update
+
+Anypoint VPN enters into asymmetric model after an on-demand maintenance if the AS paths for the two tunnels of the same VPN are different. See https://help.mulesoft.com/s/article/Anypoint-VPN-with-BGP-Routing-Enters-Asymmetric-Mode-After-update[Anypoint VPN with BGP Routing Enters Asymmetric Mode after Upgrade^] for more information.
```

Comment:
asymmetric model > an asymmetric mode

I think you mean "mode" instead of "model".
.

Reply:
Yep. Checked with my PM, and it’s "mode"